list_of_subprojects.md

```diff
@@ -6,7 +6,7 @@ At the highest level, any Github repository under any official Jupyter GitHub or
 
 The following Jupyter Subprojects have their own formal Subproject Council that elects and maintains an SSC representative:
 
-- [JupyterLab](https://github.com/jupyterlab/team-compass)
+- [Jupyter Frontends](https://github.com/jupyterlab/team-compass) and [additional link](https://github.com/jupyter/notebook-team-compass)
```

Comment:
I agree with Zach and I would like this to be corrected prior to vote.
cc @jtpio  @ericsnekbytes 

```suggestion
- [Jupyter Frontends](https://github.com/jupyterlab/team-compass)
```

Reply:
Sounds good :+1: 

Feel free to apply this change (I don't have commit rights on this repo to be able to merge this suggestion).

Reply:
@ericsnekbytes should be able to merge this as a PR owener